benchmarks/TFLite/CMakeLists.txt

```diff
@@ -158,6 +158,88 @@ set(ANDROID_MALI_GPU_TRANSLATION_FLAGS
   "--iree-vulkan-target-triple=valhall-unknown-android11"
 )
 
+set(X86_64_CASCADELAKE_CPU_TRANSLATION_FLAGS
+  "--iree-input-type=tosa"
+  "--iree-llvm-target-cpu=cascadelake"
+  "--iree-llvm-target-triple=x86_64-unknown-linux-gnu"
```

Comment:
Could prefix this name with `LINUX_`. We could also run x86_64 on Windows, and matching the naming of the `ANDROID_*` variables would be nice.

Reply:
Done.